project.gradle

```diff
@@ -50,5 +55,8 @@ if (!ext.early) {
 
         // MCMP
         deobfCompile "MCMultiPart2:MCMultiPart-exp:${mcmp_version}"
+
+        // CraftTweaker
+        compile "CraftTweaker2:CraftTweaker2-MC1120-Main:${mc_major_version}-${ct_version}"
```

Comment:
This should be 1 version string, there is absolutely no need to split it into two

Reply:
I saw that the JEI dependency used 2 versions (MC version and actual mod version) so I did the same for CT. Will change.

Reply:
Merged into 1 version with latest commit